scripts/load-test-remix.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Simulate what Thimble does when a project is remixed. This is a simplification
+ * of the code in public/editor/scripts/project/*.js, which loads a tarball,
+ * project metadata, and sometimes a default HTML file.
+ *
+ * To use this run the following command:
+ *
+ * $ node ./scripts/load-test-remix.js --url https://thimble.mozilla.org -i 1238
+ *
+ * You can specify a different --url (or -u) for staging, local dev, or prod. It will
+ * default to staging.  You must specify an --id (or -i) for the remix ID.
+ */
+
+"use strict";
+
+let program = require("commander");
+let prettyBytes = require("pretty-bytes");
+let prettyMs = require("pretty-ms");
+let request = require("request").defaults({
+    headers: {
+        "User-Agent": "Thimble Load Test"
+    },
+    time: true
+});
+
+program
+    .option("-u, --url [url]", "Host URL", "https://bramble.mofostaging.net")
+    .option("-i, --id <n>", "Remix Project ID")
+    .parse(process.argv);
+
+if(!program.url || !program.id) {
+    program.help();
+    process.exit(1);
+    return;
+}
+
+let host = program.url.replace(/\/$/, "");
+let id = program.id
+let totalTimeMS = 0;
+
+console.log("Running load test for host=%s and id=%s", host, id);
+
+/**
+ * 1) Load project tarball
+ * GET arraybuffer {host}/projects/{remixID or ID}/files/data?cacheBust={Date.now()}
```

Comment:
I'm pretty sure this is wrong. The param here is ONLY `id`. It cannot take `remixID` unless the url is suffixed with `/remix`